express_relay/sdk/python/express_relay/searcher/examples/simple_searcher_svm.py

```diff
@@ -128,19 +167,23 @@ async def evaluate_order(self, order: OrderStateAndAddress):
             [self.private_key], recent_blockhash=blockhash.blockhash
         )
         bid = BidSvm(transaction=transaction, chain_id=self.chain_id)
-        bid_id = await self.client.submit_bid(bid, False)
-        print(f"Submitted bid {bid_id} for order {order['address']}")
+        return bid
 
 
 async def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("-v", "--verbose", action="count", default=0)
-    parser.add_argument(
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
         "--private-key",
         type=str,
-        required=True,
         help="Private key of the searcher in base58 format",
     )
+    group.add_argument(
```

Comment:
What if we pass both private-key and file together? If it's hard to make sure user passed only one of them, we can add sth in help.

Reply:
these are in a `mutually_exclusive_group` defined a few lines above, so the library should reject that case automatically.